packages/p2p/src/components/modal-manager/modals/ad-cancel-modal/__tests__/ad-cancel-modal.spec.tsx

```diff
@@ -39,17 +39,29 @@ describe('<EditAdCancelModal/>', () => {
     afterAll(() => {
         document.body.removeChild(modal_root_el);
     });
-    it('should render the EditAdCancelModal', () => {
-        render(<EditAdCancelModal />);
+    it('should render the AdCancelModal', () => {
+        render(
+            <AdCancelModal
+                confirm_label="Don't cancel"
+                message='If you choose to cancel, the edited details will be lost.'
+                title='Cancel your edits?'
```

Comment:
can we store these into a variable and pass it as props since its the same as below

```suggestion
                {...mockProps}
```